guides/common/modules/proc_adding-lifecycle-environments.adoc

```diff
@@ -31,21 +31,21 @@ Note the {SmartProxy} ID of the {SmartProxy} to which you want to add a lifecycl
 [options="nowrap", subs="+quotes,verbatim,attributes"]
 ----
 # {hammer-smart-proxy} info \
---id _My_{smart-proxy-context}_ID_
+--id __My_{smart-proxy-context}_ID__
```

Comment:
```suggestion
--id __My_{smart-proxy-context-titlecase}_ID__
```
Just to make sure all of these are capitalized too?

Reply:
Yes, good call. Applied to all instances.